docs/src/test/java/jdocs/actor/ActorDocTest.java

```diff
@@ -170,6 +170,40 @@ private void receiveMsg3(Msg3 msg) {
   }
   // #optimized
 
+  public
+  // #pattern-match-java21
+  static class PatternMatchedActor extends UntypedAbstractActor {
```

Comment:
Seems like we need a `jdk-21` support, eg `java-jdk-21`. So seems we need to update the Jdk plugin then.